Using REST APIs with ADAL (ADAL C# Sample)/Program.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using System.Net.Http.Headers;
+using System.Net.Http;
+using System.Threading;
+using Newtonsoft.Json;
+using System.IO;
+
+namespace SimpleAdalConsoleApp
+{
+    class Program
+    { 
+        internal const string VSTSResourceId = "499b84ac-1321-427f-aa17-267ca6975798"; //Static value to target VSTS. Do not change
+        internal const string clientId = "872cd9fa-d31f-45e0-9eab-6e460a02d1f1"; //VS ClientId. Please use this instead of your app's clientId
+        internal const string VSTSAccountName = "myaccount"; //change to domain of your VSTS account (e.g. "myaccount" when account name is myaccount.visualstudio.com) 
+
+        static void Main(string[] args)
+        {
+            int iteration = 0;
+            
+            AuthenticationContext ctx = GetAuthenticationContext(null);
+            AuthenticationResult result = null;
+            try
+            {
+                //PromptBehavior.RefreshSession will enforce an authn prompt every time. NOTE: Auto will take your windows login state if possible
+                result = ctx.AcquireTokenAsync(VSTSResourceId, clientId, new Uri("urn:ietf:wg:oauth:2.0:oob"), new PlatformParameters(PromptBehavior.Auto)).Result;
+                Console.WriteLine("Token expires on: " + result.ExpiresOn);
+            }
+            catch (Exception ex)
+            {
+                throw ex.InnerException;
+            }
+
+            do
+            {
```

Comment:
this is a bad example, why are you looping over and over?  That was simply something I was doing as a test.  You should simplify to just call ListProjects once